extern/config.sub

```diff
@@ -1,8 +1,8 @@
 #! /bin/sh
 # Configuration validation subroutine script.
-#   Copyright 1992-2017 Free Software Foundation, Inc.
+#   Copyright 1992-2013 Free Software Foundation, Inc.
```

Comment:
Was the new code generated with an older version of autotools (or whatever it is) than the old code (ie the drop in max release date from 2017 to 2013)? Does this have any impact (since newer is surely better and shinier)?

Reply:
Good catch... I thought I was using a more recent version. I will make sure I run autoreconf one more time on the most recent tools I can when everything else checks out. Thanks.

Reply:
So this is weird. I am using autoreconf v2.69, which is the latest version, and the same I'd find on ubuntu 18.10. As I said I had checked before starting, just assumed I made a mistake somewhere. So not really sure where that is coming from. Considering I am working on CentOS there is a chance it's not the "original" 2.69 but a patched repackaged version. I'll see if I can make sense of the discrepancy.

Reply:
So, autoconf hasn't been release since 2012. I guess some other autotools were older in CentOS, so after making all the changes I just reran `autoreconf -fi` from an Ubuntu 18.10 container.